components/sidebar/sidebar.jsx

```diff
@@ -369,11 +369,12 @@ class Sidebar extends React.PureComponent {
 
     scrollToChannel = (channelId, scrollingToUnread = false) => {
         const element = $(ReactDOM.findDOMNode(this.refs[channelId]));
-        if (!element) {
+        const position = element.position();
```

Comment:
What if element is null / undefined? 

looks like it was previously checking to see if `element` was a falsey value and now it calls `position()` before checking - either the check for `element` on the next line can be dropped or its gotta be moved to before the `position()` call i think

Reply:
Nice catch, will fix